ingest/src/settings.rs

```diff
@@ -49,7 +49,7 @@ pub fn default_sink() -> String {
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "lowercase")]
 pub enum Mode {
-    Lora,
+    IOT,
```

Comment:
I'm not sure the all caps `IOT` works. I think we should just use `Iot` to avoid  enum case clippy issues

Reply:
+ 1

Reply:
Fixed d9660c6